src/nnet3bin/nnet3-combine.cc

```diff
@@ -85,23 +85,28 @@ int main(int argc, char *argv[]) {
     
     
     int32 num_nnets = po.NumArgs() - 2;
-    NnetCombiner combiner(combine_config, num_nnets, egs, nnet);
+    if (num_nnets > 1)  {
```

Comment:
I wonder if this should be
if (num_nnets > 1 || !combine_config.enforce_sum_to_one) {  
because there could actually be some meaningful change even for one nnet, if that config is set to false.